README.md

```diff
@@ -22,14 +22,50 @@ The following files are required in the folder from where the application runs:
 - **datadog_profiling_ffi.dll**: responsible for serializing and sending the profiles to Datadog via HTTP
 *Note: the corresponding .pdb files are available for debugging purposes*
 
-Before the running the application, the following environment variables **must** be set:
-- `DD_PROFILING_ENABLED` to `1`
-- `DD_API_KEY` to the Datadog API key needed to send the profiles to Datadog
-
-The others will help sorting the generated profiles:
-- `DD_VERSION` to the version of the application
-- `DD_SERVICE` to the name of the application
-- `DD_ENV` to the environment where the application runs
+## Configuration
+
+The profiler can be configured in two ways:
+
+### Option 1: Agent-based (with Datadog Agent)
+**Required environment variables:**
+- `DD_PROFILING_ENABLED=1` - Enable profiling
```

Comment:
FAR replace env var by additional parameters to `StartProfiling` or a builder

Reply:
a builder would be nice